mythril/laser/ethereum/instructions.py

```diff
@@ -1624,7 +1624,7 @@ def log_(self, global_state: GlobalState) -> List[GlobalState]:
         # TODO: implement me
         state = global_state.mstate
         dpth = int(self.op_code[3:])
-        state.stack.pop(), state.stack.pop()
+        mem_start, size = state.stack.pop(), state.stack.pop()
```

Comment:
We don’t need this change atm right?(that sonar will raise some code smells for unused code)